build.py

```diff
@@ -201,15 +201,16 @@ def get_python2_recipe():
 
 
 def get_python_packages_x86_64(python_version):
-    subprocess.run('curl https://storage.googleapis.com/travis-ci-language-archives/python/binaries/ubuntu/16.04/x86_64/python-' + python_version + '.tar.bz2 | sudo tar -xjf - --directory /', shell=True, check=True)
-
-    packages = ['autopep8', 'pylint', 'pipenv', 'ipython', 'rope', 'flake8', 'yapf']
-    patterns = {
-        '.whl': re.compile(r'(?P<package>.*)-(?P<version>.*?)-.*?-.*?-.*?\.whl'),
-        '.tar.gz': re.compile(r'(?P<package>.*)-(?P<version>.*?)\.tar\.gz'),
-        '.zip': re.compile(r'(?P<package>.*)-(?P<version>.*?)\.zip')
-    }
     with tempfile.TemporaryDirectory() as tmpdir:
+        subprocess.run('curl https://storage.googleapis.com/travis-ci-language-archives/python/binaries/ubuntu/16.04/x86_64/python-' + python_version + '.tar.bz2 | tar -xjf - --directory ' + tmpdir, shell=True, check=True)
```

Comment:
This line was for running inside Travis CI to download a specific version of Python, in order to generate the JSON recipe by Travis.  The downloaded tarball is not used after this change